### PR TITLE
fix: allow editing of event video room name

### PR DESCRIPTION
**Pull request by MeghalBisht**
as discussed in meeting

from meeting notes - Renaming event video room does not rename video room on sidepanel - Meghal

#### Short description of what this resolves:
Screenshot from 2021-03-09 22-04-08
Screenshot from 2021-03-09 22-03-56

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/components/forms/events/view/videoroom-form.hbs [hunk 1]
```diff
@@ -31,8 +31,14 @@
...
-    <label class="required">{{t 'Room'}}</label>
-    <input value={{this.roomName}} readonly>
+    {{#if this.microRoom}}
+      <label class="required">{{t 'Room'}}</label>
+      <input value={{this.microRoom}} readonly>
+    {{else}}
+      <label class="required">{{t 'Room'}}</label>
+      <Input
+        @value={{this.data.stream.name}} />
+    {{/if}}
```
Comment (iamareebjamal) [inline app/components/forms/events/view/videoroom-form.hbs:41]: It should always show stream name now, not room name

File: app/controllers/events/view/videoroom/list.js [hunk 2]
```diff
@@ -55,6 +55,7 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
   get eventColumns() {
     const { columns } = this;
     columns[0].name = this.l10n.t('Event');
+    columns[0].valuePath = 'videoStream.name';
```
Comment (iamareebjamal) [inline app/controllers/events/view/videoroom/list.js:58]: Wrong, this should remain as is

File: app/components/forms/events/view/videoroom-form.hbs (hunk 1 above)
Comment (MeghalBisht) [reply app/components/forms/events/view/videoroom-form.hbs:41]: then if microlocation rooms are renamed from wizard they dont show up renamed here

Comment (MeghalBisht) [reply app/components/forms/events/view/videoroom-form.hbs:41]: OK so now for my events, I renamed them earlier using wizard, so 2 names appear, table shows a different one and edit form shows a different one. Is it ok?

Comment (iamareebjamal) [reply app/components/forms/events/view/videoroom-form.hbs:41]: Yes

**Comment by iamareebjamal** [comment]
For making it clear to the users, add a new column to the table "Video Room Name"

**Comment by MeghalBisht** [comment]
> For making it clear to the users, add a new column to the table "Video Room Name"

OK

**Pull request merged**